chart/templates/post-delete-hook-psp.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: {{ include "rancher.fullname" . }}-post-delete
+  labels: {{ include "rancher.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": post-delete
+    "helm.sh/hook-weight": "1"
+    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded,hook-failed
+spec:
+  privileged: false
+  hostNetwork: false
+  hostIPC: false
+  hostPID: false
+  runAsUser:
+    rule: 'MustRunAsNonRoot'
```

Comment:
Correct me if I'm wrong, but isn't the user a root user?

https://github.com/rancher/rancher/blob/517013e614b0163f24d48cabe72b0f62c91595cb/chart/templates/post-delete-hook-job.yaml#L26

Reply:
@aiyengar2  Good Catch! Yes, the pod runs as a root user. 
I have fixed the PSP to fix its rule, and also update the clusterrole to use the specific PSP. 
It worked because it used other PSPs which it can pass in the cluster.